consensus/scp/src/scp_log.rs

```diff
@@ -84,10 +84,6 @@ pub struct StoredMsg<V: Value> {
 impl<V: Value, N: ScpNode<V>> LoggingScpNode<V, N> {
     /// Create a new LoggingScpNode.
     pub fn new(node: N, out_path: PathBuf, logger: Logger) -> Result<Self, String> {
-        if out_path.exists() {
```

Comment:
This is the main change. Is there a reason that this check exists?

Reply:
if we wanted to be cautious about accidentally overwriting data, could we append a unique timestamp to the temp directory name?